Setup/SetupAssist/Checks/Domain/Test-ExchangeADSetupLevel.ps1

```diff
@@ -157,11 +160,61 @@ function Test-ExchangeADSetupLevel {
 
     $adLevel = GetExchangeADSetupLevel
     $testName = "Exchange AD Latest Level"
+    $setupLog = "C:\ExchangeSetupLogs\ExchangeSetup.log"
```

Comment:
How to we determine the setup log path? If it's the system drive, it would be better to use `$env:SystemDrive` instead of the hard-coded `C:\` drive.

Reply:
Yeah, probably should use that `$env:SystemDrive`.